package.json

```diff
@@ -25,6 +26,8 @@
     "heroku-postbuild": "nuxt build examples/demo",
     "start": "nuxt start examples/demo",
     "dev": "nuxt examples/demo",
+    "postinstall": "cd examples/with-typescript && npm install",
```

Comment:
I think this is not required and also we use yarn :)

Reply:
I think that too

Reply:
yea, this is bad... oops. Maybe I'll move yarn install to dev:ts and update dev:ts to use yarn to run too